### PR TITLE
[NOT FOR MERGE] Added LogStash image to kitchen sink

### PR DESCRIPTION
**Pull request by douglaspalmer**

### CODE REVIEW

File: bin/logstash.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        component: logstash-viewer
+      podTemplate:
+        desiredState:
+          manifest:
+            version: v1beta1
+            id: logstash-server
+            containers:
+              -
+                name: logstash-image
+                image: fabric8/logstash:latest
```
Comment (jimmidyson) [inline bin/logstash.yml]: Where is this image? It's not on Docker hub - we need to get it there before we accept this PR.

File: bin/start-openshift-docker.sh [hunk 2]
```diff
@@ -192,9 +195,10 @@ getServiceIp()
 }
 
 FABRIC8_CONSOLE=http://$(getServiceIp "$K8S_SERVICES" fabric8-console)/
-DOCKER_REGISTRY=http://$(getServiceIpAndPort "$K8S_SERVICES" registry)
+DOCKER_REGISTRY=$(getServiceIpAndPort "$K8S_SERVICES" registry)
```
Comment (jimmidyson) [inline bin/start-openshift-docker.sh]: Why this line change?

Comment (douglaspalmer) [inline bin/start-openshift-docker.sh]: Sorry, this fixes a bug in the script and isn’t related to my other changes. Exporting DOCKER_REGISTRY=http://… causes 'docker push’ to fail.

Doug

Comment (jimmidyson) [inline bin/start-openshift-docker.sh]: Ah OK! Cool thanks!

**Comment by jimmidyson** [comment]
Looks like this needs a rebase as well btw @douglaspalmer.

**Comment by douglaspalmer** [comment]
Ok, done.

File: bin/start-openshift-docker.sh (hunk 2 above)
Comment (jimmidyson) [inline bin/start-openshift-docker.sh]: This has been handled by 9f63ff45d1d416327a65b12e78bc476596019336.

**Comment by jimmidyson** [comment]
Closing this now that fluentd logging is working with our contributions to kubernetes upstream.

**Pull request closed without merging**